site/en/docs/extensions/support-feedback/file-a-bug/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: 'layouts/doc-post.njk'
+title: File a bug
+description: 'TBD'
+date: 2023-10-11
+tags:
+  - extensions
+---
+
+## File a bug
+
+While developing an extension, you may find behavior that does not match the extension's documentation or is otherwise unexpected. This may be the result of a Chrome bug. If you do, please let us know by filing an appropriate issue report. Please provide enough information to reproduce the issue by following these steps:
```

Comment:
```suggestion
While developing an extension, you may find behavior that does not match the extension's documentation or is otherwise unexpected. This may be the result of a Chrome bug. Regardless, please let us know by filing an appropriate issue report. Please provide enough information to reproduce the issue by following these steps:
```